uploaded latest files in CIT 345 main project

Apy758 opened the pull request.

Apy758 committed:
uploaded latest files in CIT 345 main project

Commit:
Looks like .gitignore got renamed in the process; this explains why there are so many files in this revision.

ghost commented:
It looks like `.gitignore` got renamed to `_gitignore`, which is why there are so many changes in this revision. The conflicting files, however, **are** part of the application.

Commit:
Stopped tracking the files contained in .gitignore

Commit:
Fixed **almost** all the merge conflicts.

app/src/main/java/com/example/pete/myapplication/unlockstart.java still conflicts.

ghost commented:
I'm not sure which version of `app/src/main/java/com/example/pete/myapplication/unlockstart.java` to use. There are some pretty considerable differences between the two. @Apy758: We'll need to get together soon and go through the changes to see what should remain.
